### PR TITLE
2 aj sign up for event wpf

### PR DESCRIPTION
**Pull request by Jensen96**

### CODE REVIEW

File: Backend/Backend/BLL/UserCtrl.cs [hunk 1]
```diff
@@ -51,6 +51,11 @@ public User LogIn(DALContext ctx, string email, string clearTextPw)
             }
         }
 
+        internal bool IsRegisteredToEvent(DALContext ctx, User u, Event e)
```
Comment (refnedergaard) [inline Backend/Backend/BLL/UserCtrl.cs]: Internal??

Comment (Arcanedarkness) [inline Backend/Backend/BLL/UserCtrl.cs]: SHould not be internal

File: Frontend/Frontend/Desktop/MainWindow.xaml.cs [hunk 2]
```diff
@@ -28,7 +30,9 @@ public MainWindow()
         }
          private void Load_Window(object sender, RoutedEventArgs e)
         {
-           MainFrame.NavigationService.Navigate(new UserLogIn());
```
Comment (refnedergaard) [inline Frontend/Frontend/Desktop/MainWindow.xaml.cs]: UserLogIn is startup page.

File: Frontend/Frontend/Web/Controllers/EventController.cs [hunk 3]
```diff
@@ -43,5 +43,12 @@ public ActionResult CreateSuccess()
         {
             return View();
         }
+
+        [HttpGet]
+        public ActionResult Details(int Id)
+        {
+            
+            return View();
+        }
```
Comment (refnedergaard) [inline Frontend/Frontend/Web/Controllers/EventController.cs]: Why have you created this?

**Review by refnedergaard — COMMENTED** [review]
Added some comments that has to be edited.

**Pull request merged**